### PR TITLE
add provider link in sidebar

### LINKED ISSUE #184: add a link to the provider page in the sidebar

### PR DESCRIPTION
**Pull request by boney-bun**
fix #184

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3#184

File: django_project/core/base_templates/sidebar.html [hunk 1]
```diff
@@ -8,6 +8,13 @@
 			{{ form_data.provider_name }}
 		</div>
 		<div class='col-md-12 noplr'>
+            {% if provider.name %}
+                <ul>
+                    <li>
+                        <a href="/ford3/providers/{{ provider.id }}">{{ provider.name }}</a>
```
Comment (pierrealixt) [inline django_project/core/base_templates/sidebar.html]: See line 8 of this same file `form_data.provider_name`.
No need for a list here, a `h5` or `h6` would be enough.

**Comment by boney-bun** [comment]
there's no mockup for the provider list in the sidebar. 
so i just do improvisation.

perhaps something like this:
ford3#184revision

File: django_project/core/base_templates/sidebar.html [hunk 2]
```diff
@@ -7,6 +7,11 @@
 		<div class="col-12 side-bar-title" id="provider-name">
 			{{ form_data.provider_name }}
 		</div>
+        {% if provider.name %}
```
Comment (pierrealixt) [inline django_project/core/base_templates/sidebar.html]: please look line 8 of this file.

**Comment by Christiaanvdm** [comment]
Looks good @boney-bun : One last thing could you please remove the spaces between the campuses?
image

**Review by Christiaanvdm — APPROVED** [review]
LGTM

**Comment by boney-bun** [comment]
thanks for the feedback @Christiaanvdm @pierrealixt 
it passed travis, so i'm going to merge this PR

**Pull request merged**